dbt_project.yml

```diff
@@ -674,6 +674,8 @@ models:
         +schema: cow_protocol_ethereum
       gnosis:
         +schema: cow_protocol_gnosis
+      arbitrum:
+        +schema: cow_protocol_arbitrum
```

Comment:
we no longer needs edit to this file, we're working through cleaning out all schemas from here and applying directly in models instead for easier readability. plz do remove here 🙏